hazelcast-archunit-rules/pom.xml

```diff
@@ -0,0 +1,52 @@
+<!--
```

Comment:
Will this module be deployed to maven central? If so it needs javadoc jar. If there is no class with javadoc then we already have a workaround for some jars without any javadoc which generates an empty jar. Search for `empty-javadoc-jar`

Reply:
I think we don't have to deploy it, as it's only used during the build time. How do we control it?

Reply:
Just realized it will eventually also be used in enterprise, so needs to be deployed.

Reply:
added: https://github.com/ldziedziul/hazelcast/commit/8a7802761d56f1f07b2d2fd92dc36cec180b24a9